testing/gimmeproj/main.go

```diff
@@ -125,12 +126,19 @@ Administrative commands:
 		return fmt.Errorf("datastore.NewClient: %v", err)
 	}
 
+	// List is intended explicitly for scripting.
+	if flag.Arg(0) != "list" {
+		fmt.Println(flag.Arg(0), flag.Arg(1))
+	}
```

Comment:
remove? looks like this is for debugging?

Reply:
I have no opinions either way. The println statement itself already existed and I assumed it's intended to confirm the action being taken, since the various commands are not always transparent in what they do.

Reply:
I don't see the existing print statement.

Reply:
Done.